src/veritysetup/veritysetup.c

```diff
@@ -43,6 +46,81 @@ static int help(void) {
         return 0;
 }
 
+static int looks_like_roothashsig(const char *option) {
+        const char *val;
+        int r;
+
+        if (path_is_absolute(option)) {
+
+                r = free_and_strdup(&arg_root_hash_signature, option);
+                if (r < 0)
+                        return log_oom();
+
+                return 0;
+        }
+
+        val = startswith(option, "base64:");
+        if (val) {
+
+                r = free_and_strdup(&arg_root_hash_signature, val);
+                if (r < 0)
+                        return log_oom();
+
+                return 0;
+        }
+
+        return 1;
```

Comment:
return values appear reversed. The function names it primarily answers a boolean question (well it also cna return negative errors, but let's ignore that part), but then "yes" is returned as "0" and "no" as "1". That's unexpected... i would reverse the return codes

Reply:
Oops, indeed.